next.config.js

```diff
@@ -16,11 +16,7 @@ const webpack = (config) => {
     react: path.resolve('./node_modules/react'),
     'react-dom': path.resolve('./node_modules/react-dom'),
     next: path.resolve('./node_modules/next'),
-    'styled-components': path.resolve('./node_modules/styled-components'),
-    '@glif/base-css': path.resolve('./node_modules/@glif/base-css'),
-    '@glif/react-components': path.resolve(
-      './node_modules/@glif/react-components'
-    )
```

Comment:
Do we definitely want to remove these? I remember having them helped us link? Although that _might_ no longer be necessary after we phased out the `wallet-provider-react` package